src/components/AnonymousChat/AnonPostNotificationChannelItem.tsx

```diff
@@ -16,6 +16,7 @@ const AnonPostNotificationChannelItem: (props: MessageChannelItemProps) => React
   item,
   onChannelPressed
 }) => {
+  // TODO: Change this into useUserAuthHook
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comment indicates that `useProfileHook` should be replaced with `useUserAuthHook`. However, the change has not been made in this hunk. Please ensure to replace `useProfileHook` with `useUserAuthHook` as indicated by the TODO comment.
```diff
-   const {anonProfileId, signedProfileId} = useProfileHook();
+   const {anonProfileId, signedProfileId} = useUserAuthHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->